lib/util/uint8array_utils.js

```diff
@@ -24,9 +24,12 @@ shaka.util.Uint8ArrayUtils = class {
    * @export
    */
   static toStandardBase64(data) {
-    const bytes = shaka.util.StringUtils.fromCharCode(
-        shaka.util.BufferUtils.toUint8(data));
-    return btoa(bytes);
+    const arr = shaka.util.BufferUtils.toUint8(data);
+    if (!('toBase64' in Uint8Array)) {
```

Comment:
The "to" methods are all on Uint8Array.prototype.

Reply:
Not on plenty of TV's, but maybe we can make this a polyfill instead?